internal/elrond/elrond_soak.go

```diff
@@ -0,0 +1,63 @@
+package elrond
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/mattermost/elrond/model"
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/api"
+	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
+	pmodel "github.com/prometheus/common/model"
+	"github.com/sirupsen/logrus"
+)
+
+func checkSLOs(ring *model.Ring, thanosURL string, logger *logrus.Entry) error {
+	results, err := querySLOMetrics(ring, thanosURL, time.Now(), logger)
+	for _, result := range results {
+		if len(result) > 0 {
+			for _, rawMetric := range result {
+				sloService, ok := rawMetric.Metric["slo_service"]
+				if !ok {
```

Comment:
renaming the variable `ok` to make more sense, like `issue`, `highSLO`, `errorsSLO` so when checking `if !issue ... continue`

Reply:
So you are checking if it is invalid? so if !`invalid` continue?

Reply:
> So you are checking if it is invalid? so if !`invalid` continue?

Actually if the entry exists. Changed it to !exists ;)